src/it/java/com/zotov/edu/passportofficerestservice/GetPersonsIT.java

```diff
@@ -0,0 +1,73 @@
+package com.zotov.edu.passportofficerestservice;
+
+import com.zotov.edu.passportofficerestservice.model.PageResponse;
+import com.zotov.edu.passportofficerestservice.model.PersonSpecification;
+import org.junit.jupiter.params.ParameterizedTest;
+import org.junit.jupiter.params.provider.Arguments;
+import org.junit.jupiter.params.provider.MethodSource;
+
+import java.util.stream.Stream;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+class GetPersonsIT extends PersonBaseTest {
+
+    private Stream<Arguments> getListOfPersons() {
+        return Stream.of(
+                Arguments.of(generatePersonsPageData(100, 100, 0))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("getListOfPersons")
+    void testGetPersonsAndVerifyDefaultValues(PageResponse<PersonSpecification> expectedPersonsPage) {
+        PageResponse<PersonSpecification> pageResponse = getForPersonResponse();
+        verifyPersonsPageSizeAndNumber(expectedPersonsPage, pageResponse);
+        assertThat(pageResponse.getContent().size()).isEqualTo(expectedPersonsPage.getSize());
+    }
+
+    private Stream<Arguments> getListOfPersonsWithPageSizeAndPageNumber() {
+        return Stream.of(
+                Arguments.of(generatePersonsPageData(250, 50, 4))
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("getListOfPersonsWithPageSizeAndPageNumber")
+    void testGetPersonsAndVerify(PageResponse<PersonSpecification> expectedPersonsPage) {
+        PageResponse<PersonSpecification> pageResponse = getForPersonResponseByPassportNumber(
+                String.valueOf(expectedPersonsPage.getSize()),
+                String.valueOf(expectedPersonsPage.getNumber()));
+        verifyPersonsPageSizeAndNumber(expectedPersonsPage, pageResponse);
+        assertThat(pageResponse.getContent().size()).isEqualTo(expectedPersonsPage.getSize());
+    }
+
+    private Stream<Arguments> getListOfPersonsWithPassportNumber() {
+        return Stream.of(
+                Arguments.of(generatePersonsPageData(0, 100, 0), "NonexistentPassport")
+        );
+    }
+
+    @ParameterizedTest
+    @MethodSource("getListOfPersonsWithPassportNumber")
+    void testGetPersonsByPassportNumberAndVerify(PageResponse<PersonSpecification> expectedPersonsPage, String passportNumber) {
+        PageResponse<PersonSpecification> pageResponse = getForPersonResponseByPassportNumber(passportNumber);
```

Comment:
It seems a positive test for searching for a person by passport is missed

Reply:
The missed test has been added